### PR TITLE
FIxed duplication of states and simplified by using reflection to set…

### PR DESCRIPTION
**Pull request by Opzet**
Get startup values and names from single source via enum and data annotation with reflection.

### CODE REVIEW

**Comment by Opzet** [comment]
Missed files..oops

File: PomodoroMinimal/Model.cs [hunk 1]
```diff
@@ -146,55 +151,126 @@ public PomodoroTimer(Config config)
...
-    private string[] _activities = new string[] { "work", "short break", "long break" };
-    public string Activity => _activities[(int)_state];
-    
+
+    /// <summary>
+    /// Simplify by using reflection to set/get state from enum  ;)
+    /// Makes for easy setup
+    /// </summary>
+    [System.ComponentModel.Description("States")]
+    public enum States : Int32
```
Comment (landspersky) [inline PomodoroMinimal/Model.cs:174]: I prefer State as a single instance represents one of them

**Review by landspersky — COMMENTED** [review]
Insightful, thanks. The important data related to states are nicely brought together. I will create a new branch and merge it with some revisions.

**Comment by landspersky** [comment]
Actually, now that I think about it, reflection is big hitter for such a small detail. I agree that my state implementation is kind of cumbersome, but not worth the extra lines. If you're open to it, I would consider adding a helper class where there would be definition of the States, their default times and labels so that it is all in one place.

**Comment by Opzet** [comment]
Yes, a helper class would be nice and give you better coverage of different scenarios.  Cheers

**Pull request closed without merging**